code/client.dm

```diff
@@ -499,25 +499,14 @@ var/global/list/vpn_ip_checks = list() //assoc list of ip = true or ip = false.
 #endif
 		//Cloud data
 		if (cdn)
-			// Fetch via HTTP from goonhub
-			var/datum/http_request/request = new()
-			request.prepare(RUSTG_HTTP_METHOD_GET, "http://spacebee.goonhub.com/api/cloudsave?list&ckey=[ckey]&api_key=[config.ircbot_api]", "", "")
-			request.begin_async()
-			UNTIL(request.is_complete())
-			var/datum/http_response/response = request.into_response()
-
-			if (response.errored || !response.body)
-				logTheThing("debug", src, null, "failed to have their cloud data loaded: Couldn't reach Goonhub")
-				return
-
-			var/list/ret = json_decode(response.body)
-			if(ret["status"] == "error")
-				logTheThing( "debug", src, null, "failed to have their cloud data loaded: [ret["error"]["error"]]" )
-			else
-				cloudsaves = ret["saves"]
-				clouddata = ret["cdata"]
-				load_antag_tokens()
-				load_persistent_bank()
+			var/tries = 10
```

Comment:
```suggestion
			var/const/tries = 10
```
i don't trust DM's compiler to optimize this one bit

Reply:
It's not a constant though, it gets modified in the loop.